include/mscclpp/gpu_data_types.hpp

```diff
@@ -16,6 +16,7 @@ using __bfloat162 = __hip_bfloat162;
 #else
 
 #include <cuda_fp16.h>
+#include <cuda_runtime.h>
```

Comment:
Do we need it?

Reply:
We need `cuda_runtime_api.h`. which defined the micro CUDART_VERSION